MDVRPTW/PR11A.vrp

```diff
@@ -0,0 +1,1515 @@
+NAME: PR11A
+COMMENT: Based on Vidal et al. (2013)
+TYPE: MDVRPTW
+EDGE_WEIGHT_TYPE: EUC_2D
+DIMENSION: 364
+VEHICLES: 40
+CAPACITY: 200
+VEHICLES_MAX_DURATION: 450
+NODE_COORD_SECTION
+1	2958	4357

```

Comment:
The node coordinates are very large compared to the vehicles max duration/time windows. Is that correct?

Reply:
I had to round the X, Y coords from 3-decimal floats, so these are 1000x their original values. I should probably also round the other values to keep balance.

Reply:
Thanks!